controls/nist_ocp4.yml

```diff
@@ -13251,7 +13251,7 @@ controls:
     applicable to the configuration of OpenShift.
   rules: []
 - id: SC-6
-  status: pending
+  status: supported
```

Comment:
why supported and not automated? (Genuine question, I don't remember the difference)

Reply:
that's a good catch, it should be automated, supported means missing content automation